rallyrolebot/cogs/rally_cog.py

```diff
@@ -32,3 +32,17 @@ async def cog_command_error(self, ctx, error):
     @commands.dm_only()
     async def set_rally_id(self, ctx, rally_id):
         data.add_discord_rally_mapping(ctx.author.id, rally_id)
+
+    @commands.command(name="balance", help="View your balance")
+    @validation.is_wallet_verified()
+    async def balance(self, ctx):
+        rally_id = data.get_rally_id(ctx.message.author.id)
+        balances = rally_api.get_balances(rally_id)
+        await ctx.send(
```

Comment:
Use utils.pretty_print to preserve a common theme among RallyRoleBot outputs.

Reply:
Done.